deployment/tc510608/hook-empty-volume.json

```diff
@@ -0,0 +1,119 @@
+{
+    "kind": "DeploymentConfig",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "hooks",
+        "creationTimestamp": null,
+        "labels": {
+            "name": "mysql"
+        }
+    },
+    "spec": {
+        "strategy": {
+            "type": "Recreate",
+            "recreateParams": {
+                "pre": {
+                    "failurePolicy": "Retry",
+                    "execNewPod": {
+                        "command": [
+                             "/bin/bash",
+                             "-c",
+                             "/usr/bin/sleep 200"
+                        ],
+                        "env": [
+                            {
+                                "name": "VAR",
+                                "value": "pre-deployment"
+                            }
+                        ],
+                        "volumes": ["dataem"],
+                        "containerName": "mysql-55-centos7"
+                    }
+                },
+                "post": {
+                    "failurePolicy": "Ignore",
+                    "execNewPod": {
+                        "command": [
+                            "/bin/false"
+                        ],
+                        "env": [
+                            {
+                                "name": "VAR",
+                                "value": "post-deployment"
+                            }
+                        ],
+                        "containerName": "mysql-55-centos7"
+                    }
+                }
+            },
+            "resources": {}
+        },
+        "triggers": [
+            {
+                "type": "ConfigChange"
+            }
+        ],
+        "replicas": 1,
+        "selector": {
+            "name": "mysql"
+        },
+        "template": {
+            "metadata": {
+                "creationTimestamp": null,
+                "labels": {
+                    "name": "mysql"
+                }
+            },
+            "spec": {
+                "volumes": [
+                    {
+                        "name": "dataem",
+                        "emptyDir": {}
+                    }
+                ],
+                "containers": [
+                    {
+                        "name": "mysql-55-centos7",
+                        "image": "openshift/mysql-55-centos7:latest",
```

Comment:
I'm not sure if there is this image when you test it on ose.Could you change it to rhel image?